### PR TITLE
docs(getting started): add axure and adobe xd

### PR DESCRIPTION
**Pull request by aagonzales**
Add tabs for Axure and Adobe XD getting started tabs.

Related: https://github.com/carbon-design-system/carbon/issues/3071

### CODE REVIEW

**Comment by alisonjoseph** [comment]
You'll want to update this file with a redirect, just in case anyone is linking directly to that page https://github.com/carbon-design-system/carbon-website/blob/master/conf.d/rewrite.conf

`rewrite /get-started/design /get-started/design/sketch permanent;`

**Comment by aagonzales** [comment]
@alisonjoseph I did that but now I don't know how to resolve the conflict. Connor and I tried everything. Send help

**Comment by alisonjoseph** [comment]
🤔 All you did was change the name and add the tabs to the top, I'm not sure what would be conflicting. I can take a look in person tomorrow

**Comment by connor-leech** [comment]
@alisonjoseph @aagonzales Through trials and tribulations we triumphed. Looks like it linted our repo files but doesn't look like anything we wouldn't want done. Can you do a quick sanity check to be sure that's right?

**Comment by aagonzales** [comment]
Oh I should note that the Resource Cards don't link to the correct assets yet. I'm waiting on those to be moved but wanted to get the content up for review in the meantime. So don't merge (that label is no longer available)

File: src/pages/get-started/design/axure.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Design
+description: Rapidly build beautiful and accessible experiences. The Carbon kit contains all resources you need to get started.
+tabs: ['Sketch', 'Axure', 'Adobe XD']
+---
+
+<PageDescription>
+
+Rapidly create wireframes, interactive prototypes and documentation in Axure RP. The Carbon Axure kit contains the resources you need to get started.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
Rapidly create wireframes, interactive prototypes, and documentation in Axure RP. The Carbon Axure kit contains the resources you need to get started.
```

File: src/pages/get-started/design/axure.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Get the kit
+
+#### 1. Install Axure RP.
+
+To design with the Carbon Axure Kit you must have the latest version of [Axure RP](https://www.axure.com/download) installed.
+
+#### 2. Get the widget library.
+
+The library contains the widgets for Carbon components, design elements and other common shapes.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
The library contains the widgets for Carbon components, design elements, and other common shapes.
```

File: src/pages/get-started/design/axure.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  </ResourceCard>
+</Column>
+</Row>
+
+<br />
+<br />
+
+#### 3. Download the template.
+
+This Axure file has the settings that are required for proper display of elements.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
This Axure file has the settings that are required for the proper display of elements.
```

File: src/pages/get-started/design/axure.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+<br />
+<br />
+
+## Start designing
+
+To get started, familiarize yourself with the contents of the template and widget library.
+
+### Start with the template
+
+Create a new project document with the template instead of a blank Axure document by opening `ibm-carbon-template` and saving a copy.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
Create a new project document with the template by opening `ibm-carbon-template` and saving a copy.
```

File: src/pages/get-started/design/axure.mdx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+#### 1. Install Axure RP.
+
+To design with the Carbon Axure Kit you must have the latest version of [Axure RP](https://www.axure.com/download) installed.
+
+#### 2. Get the widget library.
+
+The library contains the widgets for Carbon components, design elements and other common shapes.
+
+Download the `.rplib` file to your computer. Open Axure and click the _Add Library_ `+` icon at the top of the _Libraries_ pane. Locate and select the `IBM Carbon 10.rplib` file.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
Download the `.rplib` file to your computer. Open Axure and click the Add Library `+` icon at the top of the Libraries pane. Locate and select the `IBM Carbon 10.rplib` file.
```

File: src/pages/get-started/design/axure.mdx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+Create a new project document with the template instead of a blank Axure document by opening `ibm-carbon-template` and saving a copy.
+
+In the template, IBM Plex Sans and IBM Plex Mono are included as web fonts in the Generate HTML settings and will be displayed properly on any computer, even without these fonts being installed locally.
+
+The _Default_, _Paragraph_ and _Text Link_ styles are pre-configured, which allows text to be pasted from the clipboard or text link inserted in a paragraph to match the typography and color guidelines by default.
+
+### Using widgets
+
+Use the widgets from the _IBM Carbon 10_ Axure library installed in step two to start designing. You'll find the widgets located in the _Libraries_ pane in the left side panel. To add a widget to the canvas, click and drag it from the Libraries pane. All widgets come with _styles_ and _interactions_ already built into them.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
Use the widgets from the IBM Carbon 10 Axure library installed in step two to start designing. You'll find the widgets located in the Libraries pane in the left side panel. To add a widget to the canvas, click and drag it from the Libraries pane. All widgets come with styles and interactions already built into them.
```

File: src/pages/get-started/design/axure.mdx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+To get started, familiarize yourself with the contents of the template and widget library.
+
+### Start with the template
+
+Create a new project document with the template instead of a blank Axure document by opening `ibm-carbon-template` and saving a copy.
+
+In the template, IBM Plex Sans and IBM Plex Mono are included as web fonts in the Generate HTML settings and will be displayed properly on any computer, even without these fonts being installed locally.
+
+The _Default_, _Paragraph_ and _Text Link_ styles are pre-configured, which allows text to be pasted from the clipboard or text link inserted in a paragraph to match the typography and color guidelines by default.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
The Default, Paragraph, and_Text Link styles are pre-configured, which allows text to be pasted from the clipboard or text link inserted in a paragraph to match the typography and color guidelines by default.
```

File: src/pages/get-started/design/axure.mdx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+
+### Using widgets
+
+Use the widgets from the _IBM Carbon 10_ Axure library installed in step two to start designing. You'll find the widgets located in the _Libraries_ pane in the left side panel. To add a widget to the canvas, click and drag it from the Libraries pane. All widgets come with _styles_ and _interactions_ already built into them.
+
+To edit the text on a widget, double-click it or select it and press `ENTER`.
+
+To change style properties of your widgets, use the _Style_ pane on the right-hand side of the app or use the _Style Toolbar_ at top of the canvas.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
To change style properties of your widgets, use the Style pane on the right-hand side of the app or use the Style Toolbar at top of the canvas.
```

File: src/pages/get-started/design/axure.mdx [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+In an Axure RP document, styles already in use in the document prevail when adding widgets from a library. The existing style in the document will be applied to a library widget when the widget is dragged onto the document canvas.
+
+Replacing the library won’t update the widget styles in your existing project files automatically. Follow the manual process below to update the styles.
+
+Import the styles from the library file into project documents using the _Import Wizard._
+
+1. Open your project in Axure RP.
+2. Go to `file` → `Import from RP file ...` and select the .rplib library file.
+3. In the wizard, go to the _Import Widget Styles_ step, and select all custom styles. "Default" should **not** be selected.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
3. In the wizard, go to the Import Widget Styles step, and select all custom styles. "Default" should **not** be selected.
```

File: src/pages/get-started/design/axure.mdx [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+<Row>
+<Column colLg={8}>
+
+![Axure wizard settings](images/axure-updates.png)
+
+</Column>
+</Row>
+
+4. Skip to the end of the wizard and click _Finish_ to import the styles.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
4. Skip to the end of the wizard and click Finish to import the styles.
```

File: src/pages/get-started/design/axure.mdx [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
+If you're an IBMer, join the #carbon-axure-kit Slack channel to get notified about changes.
+
+### Updating existing documents
+
+In an Axure RP document, styles already in use in the document prevail when adding widgets from a library. The existing style in the document will be applied to a library widget when the widget is dragged onto the document canvas.
+
+Replacing the library won’t update the widget styles in your existing project files automatically. Follow the manual process below to update the styles.
+
+Import the styles from the library file into project documents using the _Import Wizard._
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
Import the styles from the library file into project documents using the Import Wizard.
```

File: src/pages/get-started/design/axure.mdx [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+
+To get started, familiarize yourself with the contents of the template and widget library.
+
+### Start with the template
+
+Create a new project document with the template by opening `ibm-carbon-template` and saving a copy.
+
+In the template, IBM Plex Sans and IBM Plex Mono are included as web fonts in the Generate HTML settings and will be displayed properly on any computer, even without these fonts being installed locally.
+
+The Default, Paragraph, and_Text Link styles are pre-configured, which allows text to be pasted from the clipboard or text link inserted in a paragraph to match the typography and color guidelines by default.
```
Comment (janchild) [inline src/pages/get-started/design/axure.mdx]:
```suggestion
The Default, Paragraph, and Text Link styles are pre-configured, which allows text to be pasted from the clipboard or text link inserted in a paragraph to match the typography and color guidelines by default.
```

**Comment by aagonzales** [comment]
Ok links have been updated and are now working!

**Pull request merged**